packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -173,6 +182,11 @@ export class UnfurlService {
                 Logger.warn(
                     `Retrying (${retries}) to fetch screenshots from headless chrome ${e.message}`,
                 );
+                const delay = (ms: number) =>
+                    new Promise((res) => {
+                        setTimeout(res, ms);
+                    });
+                await delay(10000); // Wait 10 seconds before retrying
```

Comment:
Wait 10 seconds before retrying, so if the browser is killed, we give it some time to restart . 

Currently all retries fail immediatelly.

![image](https://user-images.githubusercontent.com/1983672/221131019-efcbbd17-00c4-47a1-8b15-a204da913a2b.png)

Reply:
If we ran this as a job in the worker we wouldn't need our own logic for this!

Reply:
I agree, but first we need to fix this https://github.com/lightdash/lightdash/issues/4583

Reply:
Added a comment in that ticket to remove this retrying logic once the jobs are retrying properly.